sdk/template/template/README.md

```diff
@@ -75,16 +75,27 @@ Create a section for each top-level service concept you want to explain.
 
 Create several code examples for how someone would use your library to accomplish a common task with the service.
 
+```js snippet:FooBar
```

Comment:
Do we want to keep these here or have real samples that are related to the `template` project?

Reply:
This will be removed and replaced with something more realistic.